CHANGELOG/CHANGELOG-1.2.md

```diff
@@ -0,0 +1,36 @@
+# v1.2.0 - Changelog since v1.1.0
+
+## Features
+
+- Improved Windows Support
+  - Switch to CSI Proxy v1beta2 API (previously v1beta1). ([#669](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/669), [@jingxu97](https://github.com/jingxu97))
```

Comment:
To clarify, this is for driver deployment, it updates named pipe path in node.yaml under deployment to use csi proxy disk v1beta2. I think this might belong to Tests section.